src/components/intro/intro.css

```diff
@@ -21,4 +21,29 @@
 .logoContainer {
   height: 30vw;
   width: 30vw;
+  margin-right: 6%;
+
+}
+
+@media only screen and (max-width: 768px) {
+  /* For mobile phones: */
+  .logoContainer {
+    height: 45vw;
+    width: 45vw;
+    margin-right: 0%;
+  }
+  .introContainer {
+    height: 100vh;
+    width: 100%;
+    background-color: #f0f0e9;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    background-image: radial-gradient(
```

Comment:
Can you explain me what this is doing. I'm not saying that is wrong, just that I'd like to understand its function.